roles/pulp_common/tasks/main.yml

```diff
@@ -32,6 +32,16 @@
   loop_control:
     loop_var: __pulp_common_req_var
 
+- name: Check required roles if FIPS detected
+  assert:
+    that:
+      - "'pulp_devel' in role_names"
+    fail_msg: >
+      Pulp does not officially support FIPS since Django (a dependency) is not FIPS
+      compatible. However, if you still wish to run Pulp with FIPS please add the 'pulp_devel'
+      role to your playbook to enable patching of the Django installation.
```

Comment:
I believe we need to remove this second sentence entirely because devel installs are totally unsupported. We're basically pulling FIPS support entirely for end users with this release.